### PR TITLE
Benchmark size of protocol primitives

### LINKED ISSUE #47: Benchmark size of protocol primitives
- [x] Benchmarks size of PVSS transcript for 2, ..., 64 shares 
- See [previous work](https://github.com/nucypher/ferveo/pull/14/files#diff-bda7e0020fcebb60a3ece40d383db69db8476f790c18f68f3b7bf268def6d6e6)

### PR DESCRIPTION
**Pull request by piotr-roslaniec**
- Closes #47 

<details>
<summary>Benchmark results</summary>

<h1>Size of PVSS transcripts </h1>

| shares_num | threshold | pvss_transcript_size_bytes |
| ---------- | --------- | -------------------- |
| 2          | 2         | 400                  |
| 4          | 3         | 640                  |
| 4          | 4         | 688                  |
| 8          | 5         | 1120                 |
| 8          | 6         | 1168                 |
| 8          | 7         | 1216                 |
| 8          | 8         | 1264                 |
| 16         | 9         | 2080                 |
| 16         | 11        | 2176                 |
| 16         | 13        | 2272                 |
| 16         | 16        | 2416                 |
| 32         | 17        | 4000                 |
| 32         | 22        | 4240                 |
| 32         | 26        | 4432                 |
| 32         | 32        | 4720                 |
| 64         | 33        | 7840                 |
| 64         | 43        | 8320                 |
| 64         | 52        | 8752                 |
| 64         | 64        | 9328                 |

</details>

### CODE REVIEW

File: ferveo/examples/bench_primitives_size.rs [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    shares_num: u32,
+) -> PubliclyVerifiableDkg<EllipticCurve> {
+    let keypairs = gen_keypairs(shares_num);
+    let validators = gen_validators(&keypairs);
+    let me = validators[validator].clone();
+    PubliclyVerifiableDkg::new(
+        validators,
+        Params {
+            tau: 0,
+            security_threshold: shares_num / 3,
```
Comment (cygnusv) [inline ferveo/examples/bench_primitives_size.rs:65]: Threshold being N/3 of N is not a realistic setting. We want at the very least N/2 + 1, and 2N/3 is the most likely. 
How about we parametrize this to:  N/2 + 1 (~51%), 2N/3 (~66%), 4N/5 (80%), N (100%)

Comment (piotr-roslaniec) [reply ferveo/examples/bench_primitives_size.rs:65]: Sounds good, looking into it.

**Comment by piotr-roslaniec** [comment]
@cygnusv The updated benchmark numbers should make more sense now

**Review by theref — APPROVED** [review]
What are the units of `pvss_transcript_size`?

**Comment by piotr-roslaniec** [comment]
@theref It's expressed in bytes (`u8`). I've updated the naming.

**Pull request merged**